installer/install-golangci-lint-langserver.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -e
+
+"$(dirname "$0")/go_install.sh" github.com/nametake/golangci-lint-langserver@latest
+"$(dirname "$0")/go_install.sh" github.com/golangci/golangci-lint@latest
```

Comment:
Same above https://github.com/golangci/golangci-lint#binary

Reply:
golangci-lint-langserver does not support Windows binary.